code/API_definitions/device-status.yaml

```diff
@@ -344,6 +385,29 @@ components:
       description: Roaming status. True, if it is roaming
       type: boolean
 
+    ConnectivityStatusResponse:
+      type: object
+      required:
+        - connected
+      properties:
+        connected:
```

Comment:
Maybe connectionStatus or connectivityStatus would be better, it's strange otherwise when connected=NOT_CONNECTED 🙂